app/mailers/notify.rb

```diff
@@ -8,7 +8,9 @@ class Notify < ApplicationMailer
   def monthly_report_registration(user_id, monthly_report_id)
     @report = MonthlyReport.find(monthly_report_id)
     @user = User.find(user_id)
+    return unless @user.group
+
     @title = "#{@user.name}が#{@report.target_month.strftime('%Y年%m月')}の月報を登録しました"
-    mail(to: @user.email, subject: @title)
+    mail(to: @user.group.email, subject: @title)
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)

<!-- Comment by [SideCI](https://sideci.com) -->